public/src/js/widgets/fetch-latest-packages.js

```diff
@@ -0,0 +1,31 @@
+import Extension from 'models/extension';
+import {request} from 'modules/authed-ajax';
+import CONST from 'constants/defaults';
+
+export default class extends Extension {
+    constructor(baseModel) {
+        super(baseModel);
+        this.baseModel = baseModel;
+        this.fetchPackages();
+        this.pollPackages();
+    }
+
+    fetchPackages() {
+        var that = this;
+        return request({
+            url: CONST.apiBase + '/story-packages/latest'
+        })
+        .then(response => {
+            that.baseModel.latestPackages(response.results);
+        })
+        .catch(function () {
+            throw new Error('latest packages endpoint is invalid or unavailable');
```

Comment:
The `catch` should not throw again, we can use the yellow alert bar for this, just like latest search